istioctl/cmd/describe.go

```diff
@@ -1041,36 +1053,38 @@ the configuration objects that affect that service.`,
 
 			writer := cmd.OutOrStdout()
 
-			pods, err := client.CoreV1().Pods(ns).List(context.TODO(), metav1.ListOptions{})
-			if err != nil {
-				return err
+			labels := ""
+			for k, v := range svc.Spec.Selector {
+				labels += fmt.Sprintf("%s=%s,", k, v)
```

Comment:
how about to define labels as a string array, then use `strings.Join()` to complete such splice? then it's unnecessary for you to trim the separator in tail in L#1060 ?

Reply:
@zhlsunshine Thanks, fixed.